coldfront/core/project/forms_/new_project_forms/request_forms.py

```diff
@@ -243,6 +245,23 @@ def clean_email(self):
 
         return email
 
+class SavioProjectPIDepartmentForm(forms.Form):
+    '''Form prompting for the departments of a new PI if one is not found
+    through LDAP. Has user select one or more from the dozens of departments
+    present in Department.objects.all()'''
+
+    departments = forms.ModelMultipleChoiceField(
+        label='Departments',
+        queryset=Department.objects.order_by('name').all(),
+        required=True
+    )
+
+    def clean_departments(self):
```

Comment:
Doesn't the `required=True` flag already require that one be given?

Reply:
Yeah, I removed the method, thanks